Web/View/LandingPages/Show.hs

```diff
@@ -16,30 +15,49 @@ instance View ShowView where
             <a href={EditLandingPageAction landingPage.id} class="text-blue-500 text-sm hover:underline hover:text-blue-600">(Edit)</a>
         </div>
 
-        {orderAndRenderParagraphs landingPage.paragraphCtasLandingPages landingPage.paragraphQuotes}
+        {orderAndRenderParagraphs landingPageWithRecords }
 
     |]
         where
+            landingPage = landingPageWithRecords.landingPageWithRecordsLandingPage
+
             breadcrumb = renderBreadcrumb
                             [ breadcrumbLink "LandingPages" LandingPagesAction
                             , breadcrumbText "Show LandingPage"
                             ]
 
 
-orderAndRenderParagraphs :: (?context::ControllerContext) => [ParagraphCta] -> [ParagraphQuote] -> Text.Blaze.Html.Html
-orderAndRenderParagraphs ctas quotes =
+orderAndRenderParagraphs :: (?context::ControllerContext) => LandingPageWithRecords -> Html
+orderAndRenderParagraphs landingPageWithRecords =
     [hsx|{forEach allSorted (\rendered -> rendered)}|]
     where
-        ctas' = ctas
+        paragraphCtas = landingPageWithRecords.landingPageWithRecordsParagraphCtas
+        paragraphQuotes = landingPageWithRecords.landingPageWithRecordsParagraphQuotes
+        paragraphCtaRefLandingPages = landingPageWithRecords.landingPageWithRecordsParagraphCtaRefLandingPages
+
+        ctas = paragraphCtas
             |> fmap (\paragraph ->
+                let
+                    -- Get the referenced Landing page out of the ParagraphCTA, through the `ParagraphCtaRefLandingPageId`
+                    -- property.
+                    refLandingPageButton =
+                            paragraphCtaRefLandingPages
+                                -- Get the referenced Landing page
+                                |> filter (\paragraphCtaRefLandingPage -> paragraphCtaRefLandingPage.id == paragraph.refLandingPageId)
+                                |> head
```

Comment:
`filter` => `head` can also be done written as just `find`

```suggestion
                                |> find (\paragraphCtaRefLandingPage -> paragraphCtaRefLandingPage.id == paragraph.refLandingPageId)
```

Reply:
>  Wouldn't landingPage work as well instead of landingPageWithRecordsLandingPage?

I started with `landingPage`, hoping for the best. But the compiler complained about `ambiguous` on the controller as we also have `landingPage` [here](https://github.com/amitaibu/ihp-landing-page/blob/d482970a7e4959790094f3dd73e91a729a674746/Web/View/LandingPages/New.hs#L4).

It would be much nicer to not use such long names. so I wonder if there's a better way

Reply:
This shouldn't be a problem anymore with dot notation. I just tried locally and got everything to compile with the shorter names (if you can give me access to the repo I could push it)

Reply:
> if you can give me access to the repo I could push it

Done

Reply:
Thx, just pushed my changes